.github/workflows/rust.yml

```diff
@@ -58,15 +58,12 @@ jobs:
     - uses: actions/checkout@v3
     - uses: stellar/actions/rust-check-git-rev-deps@main
 
-  semver-checks:    
+  semver-checks:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
     - run: rustup update
-    - uses: stellar/binaries@v40
-      with:
-        name: cargo-semver-checks
-        version: 0.42.0
+    - run: cargo install cargo-semver-checks --locked
```

Comment:
I don't think we should do this change. If there is a semver update necessary, we should update it in the binaries repo for everyone, like here: https://github.com/stellar/binaries/pull/43